exchanges/kraken.js

```diff
@@ -203,8 +203,12 @@ Trader.prototype.getPortfolio = function(callback) {
   this.kraken.api('Balance', {}, _.bind(setBalance, this));
 };
 
+// This assumes that only limit orders are being placed with standard assets pairs
+// It does not take into account volume discounts.
+// Base maker fee is 0.16%, taker fee is 0.26%.
```

Comment:
Gekko should always be making (unless the price moves hard between a ticker fetch and order placement, which on Kraken I would not rule out).